cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModel.kt

```diff
@@ -63,6 +63,7 @@ interface CampaignMessage {
 internal fun CampaignType.toCategoryId() = when (this) {
     CampaignType.GDPR -> 1
     CampaignType.CCPA -> 2
+    CampaignType.USNAT -> 6
```

Comment:
@carmelo-iriti how are we sure that `categoryId` for `UsNat` is `6`?

Reply:
Correct